android/quest/src/test/assets/configs/app/measure_report_config.json

```diff
@@ -2,6 +2,7 @@
   "appId": "app",
   "configType": "measureReport",
   "registerId": "ancRegister",
+  "registerDate": "2020-10-27",
```

Comment:
@f-odhiambo Are we suppose to refer to this as `registerDate`?

Reply:
This is the date which will be used as campaign register date,